ruby/hangman/hangman.rb

```diff
@@ -18,7 +17,7 @@ def is_game_over
 	end
 
 	def add_guess_count
```

Comment:
are you using this method outside of the class?  if not it could be private as well